.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Anurima\\AppData\\Local\\Continuum\\anaconda3\\python.exe"
+}
```

Comment:
Add this file in ```.gitignore``` as we don't want each other's vscode's settings.json files.

Reply:
This I am not sure about but I have now assigned you to rectify if there is any more issue.